src/components/button/themes/button/shared/button.indigo.scss

```diff
@@ -42,6 +48,10 @@ $outlined-theme: $indigo-outlined;
 
     &:focus {
         border-color: var-get($outlined-theme, 'focus-border-color');
+
+        &:hover {
+            border-color: var-get($fab-theme, 'hover-border-color');
```

Comment:
Are you using the border color from `$fab-theme` instead of `$outlined-theme` on purpose?

Reply:
no, thank you for the comment